config/development.json

```diff
@@ -29,6 +29,7 @@
 	"rebrand_cities_prefix": "rebrandcitiesdevelopmentsite",
 	"livechat_support_locales": [ "en", "es", "pt-br" ],
 	"features": {
+		"wordpress-action-search": false,
```

Comment:
Two things for this:

1. Feature flags are sorted in alphabetical order, so let's move this to the w-initial section.
2. When we add a new feature flag, it's better to be available across all the environmental configs so it's explicit even if it means to be disabled by default. So let's also put it in horizon.json, wpcalypso.json, production.json, test.json, desktop-development.json, desktop.json, and stage.json.

Reply:
Got it thanks was wondering whether to do this but was not sure what the prevailing practice was 👍